common/djangoapps/student/views.py

```diff
@@ -1238,12 +1253,13 @@ def login_user(request, error=""):  # pylint: disable=too-many-statements,unused
         return set_logged_in_cookies(request, response, user)
 
     if settings.FEATURES['SQUELCH_PII_IN_LOGS']:
-        AUDIT_LOG.warning(u"Login failed - Account not active for user.id: {0}, resending activation".format(user.id))
+        AUDIT_LOG.warning(u"Login failed - Account not active for user.id: %s, resending activation", user.id)
     else:
-        AUDIT_LOG.warning(u"Login failed - Account not active for user {0}, resending activation".format(username))
+        AUDIT_LOG.warning(u"Login failed - Account not active for user %s, resending activation", username)
 
     reactivation_email_for_user(user)
-    not_activated_msg = _("This account has not been activated. We have sent another activation message. Please check your email for the activation instructions.")
+    not_activated_msg = _("This account has not been activated. We have sent another activation "
+                          "message. Please check your email for the activation instructions.")
```

Comment:
I didn't do that because we had some weird i18n issue around doing this before. I can't quite remember. @nedbat ?